src/cplus_plugin/gui/metrics_builder_dialog.py

```diff
@@ -407,26 +410,69 @@ def initialize_collection(self):
         """Creates an initial profile collection. Use this if None is
         specified.
         """
-        # Add a default area column
+        # Add a default columns
+        defaults_columns = []
+        # Area
         area_metric_column = MetricColumn.create_default_column(
             self.AREA_COLUMN, tr("Area (Ha)"), f"@{VAR_ACTIVITY_AREA}"
         )
         area_metric_column.auto_calculated = True
         area_metric_column.format_as_number = True
+        defaults_columns.append(area_metric_column)
+
+        # Carbon protect
```

Comment:
It seems to be repetitive code until NPV. Rather create a function e.g.
```
def _add_numeric_column(columns, title, short_title, expression):
    column = MetricColumn.create_default_column(
        title,
        short_title,
        expression,
    )
    column.auto_calculated = False
    column.format_as_number = True
    columns.append(column)
```

Then (depending on where you create the function)
```
_add_numeric_column(
    defaults_columns,
    tr("Carbon Impact Protect"),
    tr("C.I. Protect"),
    f"{FUNC_CARBON_IMPACT_PROTECT}()",
)
```